src/test/kotlin/org/pac4j/vertx/handler/impl/Functions.kt

```diff
@@ -65,7 +64,7 @@ fun logoutHandler(vertx: io.vertx.core.Vertx): Handler<RoutingContext> {
 }
 
 
-fun getProfileManager(rc: RoutingContext): VertxProfileManager<TestOAuth1Profile> {
```

Comment:
We shouldn't release yet by the way, the API changed a very small amount (if someone wants to consume web contexts/vertx profile managers) in this PR, so the API is not quite stable enough to issue a new release in my view.